docs/getting-started.md

```diff
@@ -72,20 +72,33 @@ helm repo add antrea https://charts.antrea.io
 helm repo update
 ```
 
+To install Flow Aggregator, please run the following command:
+
+```bash
+helm install flow-aggregator antrea/flow-aggregator --set clickHouse.enable=true,recordContents.podLabels=true -n flow-aggregator --create-namespace
+```
+
 To enable both [Grafana Flow Collector](network-flow-visibility.md) and
 [NetworkPolicy Recommendation](networkpolicy-recommendation.md), please install
-Flow Aggregator and Theia by runnning the following commands:
+Theia by runnning the following commands:
 
 ```bash
-helm install flow-aggregator antrea/flow-aggregator --set clickHouse.enable=true,recordContents.podLabels=true -n flow-aggregator --create-namespace
 helm install theia antrea/theia --set sparkOperator.enable=true -n flow-visibility --create-namespace
 ```
 
-To enable only Grafana Flow Collector, please install Flow Aggregator and Theia
-by runnning the following commands:
+From Theia v0.3, [Theia Command-line Tool](theia-cli.md) uses Theia Manager
+as a layer to connect and manage other resources like ClickHouse and Spark
+Operator. To enable Theia Manager, please install Theia by runnning the
```

Comment:
Do we need to set theiaManager.enable=true as we already set it to true in values.yaml?

Reply:
Actually the answer is no. It is not necessary when we set the default to true, while it is no harm to add this. I just have not come up with a better description to indicate only for v0.3 and v0.4 we need to add `theiaManager.enable=true`.